plugins/puppetdb/README.md

```diff
@@ -0,0 +1,44 @@
+# PuppetDB Plugin
+
+A frontend plugin to integrate PuppetDB with Backstage. When combined with the
+[catalog-backend-module-puppetdb](https://github.com/backstage/backstage/blob/master/plugins/catalog-backend-module-puppetdb/README.md) plugin, this
+frontend plugin allows viewing PuppetDB reports, including their logs and events, of Backstage resource entities.
+
+![Reports list](https://raw.githubusercontent.com/backstage/backstage/42b65232e763d3e39e2e641b105d2ad469db7a59/plugins/puppetdb/assets/Reports.png)
+
+## Getting started
+
+To get started, you need a running instance of PuppetDB. You can find instructions on how to install it
+[here](https://www.puppet.com/docs/puppetdb/7/install_via_module.html).
+
+### Installation
+
+1. Install the plugin with `yarn` in the root of your Backstage application directory:
+
+```bash
+yarn --cwd packages/app add @backstage/plugin-puppetdb
+```
+
+1. Import and use the plugin in `packages/app/src/App.tsx`:
+
+```tsx
+import { PuppetDbPage } from '@backstage/plugin-puppetdb';
+
+const routes = (
+  <FlatRoutes>
+    {/* ...other routes */}
+    <Route path="/puppetdb" element={<PuppetDbPage />} />
+  </FlatRoutes>
+);
+```
+
+### Configuration
```

Comment:
My Puppet experience is limited and we used Foreman so I may be mixing things up but should we include a link to documentation about authenticating against the API? Is it normal to have this wide open? I guess I would expect to seem some auth header with a token of some sort in this example.

Reply:
PuppetDB supports only certificate based or no authentication at all. Certificate based authentication might be tricky to implement here, I guess, so I would like to leave if for the future if anyone needs it.

For now, I've added the following to the README:
> The PuppetDB [should be configured](https://www.puppet.com/docs/puppetdb/7/configure.html#host) to allow being accessed from your Backstage instance.

Reply:
OK, that's fine with me. 👍